modular_skyrat/modules/marines/code/clothing/spacesuits/miscellaneous.dm

```diff
@@ -0,0 +1,33 @@
+/obj/item/clothing/head/helmet/space/hardsuit/ert/marine
+	name = "marine helmet"
+	desc = "The integrated helmet of an advanced Marine hardsuit, fielded by Nanotrasen's navy."
+	icon = 'modular_skyrat/master_files/icons/obj/clothing/hats.dmi'
+	worn_icon = 'modular_skyrat/master_files/icons/mob/clothing/head.dmi'
+	icon_state = "hardsuit0-marine"
+	inhand_icon_state = "hardsuit0-marine" //should never be able to hold it but hey
+	hardsuit_type = "marine"
+	armor = list(MELEE = 65, BULLET = 65, LASER = 65, ENERGY = 60, BOMB = 80, BIO = 100, FIRE = 100, ACID = 80)
+
+/obj/item/clothing/suit/space/hardsuit/ert/marine
+	name = "marine hardsuit"
+	desc = "The standard issue hardsuit of the Nanotrasen Marine Corps, fitted with advanced plasteel composite plating and a nanogel-lined undersuit for maximum protection and minimal slowdown."
+	icon = 'modular_skyrat/master_files/icons/obj/clothing/suits.dmi'
+	worn_icon = 'modular_skyrat/master_files/icons/mob/clothing/suit.dmi'
+	icon_state = "marinesuit"
+	inhand_icon_state = "ert_security"
+	helmettype = /obj/item/clothing/head/helmet/space/hardsuit/ert/marine
+	armor = list(MELEE = 65, BULLET = 65, LASER = 65, ENERGY = 60, BOMB = 80, BIO = 100, FIRE = 100, ACID = 80)
```

Comment:
(Just so anyone reviewing this can get the reference easier, base hardsuit/ert stats are
`(MELEE = 65, BULLET = 50, LASER = 50, ENERGY = 60, BOMB = 50, BIO = 100, FIRE = 80, ACID = 80)`
)

Reply:
Indeed, the marines ((when added)) should never actually be going to the station lest admins spawn the suit, so it’s more focused for actual heavy combat as a marine might